src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/ImportOrchestratorJob.cs

```diff
@@ -264,9 +272,33 @@ private async Task ValidateResourcesAsync(ImportOrchestratorJobDefinition inputD
             });
         }
 
-        private async Task SendImportMetricsNotification(JobStatus jobStatus, JobInfo jobInfo, ImportOrchestratorJobResult currentResult, ImportMode importMode)
+        private async Task SendImportMetricsNotification(JobStatus jobStatus, JobInfo jobInfo, ImportOrchestratorJobResult currentResult, ImportMode importMode, FhirRequestContext fhirRequestContext)
         {
             _logger.LogInformation("SucceededResources {SucceededResources} and FailedResources {FailedResources} in Import", currentResult.SucceededResources, currentResult.FailedResources);
+
+            if (importMode == ImportMode.IncrementalLoad)
+            {
+                var incrementalImportProperties = new Dictionary<string, string>();
+                incrementalImportProperties["JobId"] = jobInfo.Id.ToString();
+                incrementalImportProperties["SucceededResources"] = currentResult.SucceededResources.ToString();
+                incrementalImportProperties["FailedResources"] = currentResult.FailedResources.ToString();
+
+                _auditLogger.LogAudit(
+                AuditAction.Executed,
+                operation: ImportMode.IncrementalLoad.ToString(),
+                resourceType: string.Empty,
+                requestUri: fhirRequestContext.Uri,
+                statusCode: HttpStatusCode.Accepted,
+                correlationId: fhirRequestContext.CorrelationId,
+                callerIpAddress: string.Empty,
+                callerClaims: null,
+                customHeaders: null,
+                operationType: string.Empty,
+                callerAgent: DefaultCallerAgent,
+                additionalProperties: incrementalImportProperties);
+                _logger.LogInformation("After logging Audit logs for incremental import");
```

Comment:
nit: can we reword the log statement

Reply:
Are you ok with "Audit logs for incremental import are added"?

Reply:
Done